app/models/sub_service_request.rb

```diff
@@ -320,7 +320,7 @@ def is_locked?
   def can_be_edited?
     if organization.has_editable_statuses?
       self_or_parent_id = find_editable_id(self.organization.id)
-      EDITABLE_STATUSES[self_or_parent_id].include?(self.status)
+      EDITABLE_STATUSES[self_or_parent_id].include?(self.status) && !is_complete?
```

Comment:
`complete` is an editable status for a couple Organizations, so this shouldn't be here.